### PR TITLE
AST-2087 - Customizer responsive toggle control ui changes

### PR DESCRIPTION
**Pull request by er-roshan**
### Description
AST-2087 - Changed Customizer responsive toggle control ui

### Screenshots
https://d.pr/6Hgmeu

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/base/assets/js/customizer-preview.js [hunk 1]
```diff
@@ -690,6 +690,35 @@ function astra_builder_visibility_css( section, selector, default_property = 'fl
...
+			dynamicStyle += '} ';
+			dynamicStyle += '} ';
+
+			astra_add_dynamic_css( section + '-visibility-responsive', dynamicStyle );
+		} );
+
+	} );
+
 	// Header Desktop visibility.
 	wp.customize( 'astra-settings[' + section + '-hide-desktop]', function( setting ) {
```
Comment (imnavanath) [inline inc/builder/type/base/assets/js/customizer-preview.js:723]: Assuming we are migrating users from old control to new so do we require this old code anymore??

File: inc/customizer/configurations/builder/class-astra-builder-base-configuration.php [hunk 2]
```diff
@@ -224,53 +233,37 @@ public static function prepare_typography_options( $section_id, $required_condit
...
-				'name'      => ASTRA_THEME_SETTINGS . '[' . $_section . '-hide-desktop]',
+				'name'      => ASTRA_THEME_SETTINGS . '[' . $_section . '-visibility-responsive]',
+				'default'   => astra_get_option(
+					'' . $_section . '-visibility-responsive',
+					array(
+						'desktop' => ! isset( $astra_options[ '' . $_section . '-visibility-responsive' ] ) && isset( $astra_options[ '' . $_section . '-hide-desktop' ] ) ? 0 === $astra_options[ '' . $_section . '-hide-desktop' ] ? 1 : 0 : 1,
+						'tablet'  => ! isset( $astra_options[ '' . $_section . '-visibility-responsive' ] ) && isset( $astra_options[ '' . $_section . '-hide-tablet' ] ) ? 0 === $astra_options[ '' . $_section . '-hide-tablet' ] ? 1 : 0 : 1,
+						'mobile'  => ! isset( $astra_options[ '' . $_section . '-visibility-responsive' ] ) && isset( $astra_options[ '' . $_section . '-hide-mobile' ] ) ? 0 === $astra_options[ '' . $_section . '-hide-mobile' ] ? 1 : 0 : 1,
+					)
+				),
```
Comment (imnavanath) [inline inc/customizer/configurations/builder/class-astra-builder-base-configuration.php:254]: If we mapped defaults into customizer-defaults file then no need to add condition here again.

File: inc/builder/type/base/assets/js/customizer-preview.js (hunk 1 above)
Comment (er-roshan) [reply inc/builder/type/base/assets/js/customizer-preview.js]: I have removed the old codes.

File: inc/customizer/configurations/builder/class-astra-builder-base-configuration.php (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/configurations/builder/class-astra-builder-base-configuration.php:256]: Also nested condition without parenthesis is deprecated in PHP 7.4 https://share.bsf.io/NQu4rA8Q

Comment (er-roshan) [reply inc/customizer/configurations/builder/class-astra-builder-base-configuration.php]: Solved

**Review by imnavanath — APPROVED** [review]

**Pull request merged**